app/models/token/expirable_token.rb

```diff
@@ -64,30 +64,34 @@ def valid_plaintext?(input)
       end
 
       def expired?
-        expires_on.nil? || Time.now > expires_on
+        expires_on.nil? || Time.current > expires_on
       end
 
-      def validity_time
-        self.class.validity_time
-      end
+      delegate :validity_time, to: :class
```

Comment:
:green_circle: Ordering-wise I'd move delegations to the top, so that it's not hidden between multi-line method definitions.

E.g. right next to `after_save`.